crates/bitwarden-exporters/resources/json_export.json

```diff
@@ -141,6 +141,26 @@
       "revisionDate": "2024-01-30T17:54:50.706Z",
       "creationDate": "2024-01-30T17:54:50.706Z",
       "deletedDate": null
+    },
+    {
+      "id": "646594a9-a9cb-4082-9d57-0024c3fbcaa9",
+      "folderId": null,
+      "organizationId": null,
+      "collectionIds": null,
+      "name": "My ssh key",
+      "notes": null,
+      "type": 5,
+      "sshKey": {
+        "privateKey": "-----BEGIN OPENSSH PRIVATE KEY-----\nb3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAAAMwAAAAtzc2gtZW\nQyNTUxOQAAACBinNE5chMtCHh3BV0H1+CpPlEQBwR5cD+Xb9i8MaHGiwAAAKAy48fwMuPH\n8AAAAAtzc2gtZWQyNTUxOQAAACBinNE5chMtCHh3BV0H1+CpPlEQBwR5cD+Xb9i8MaHGiw\nAAAEAYUCIdfLI14K3XIy9V0FDZLQoZ9gcjOnvFjb4uA335HmKc0TlyEy0IeHcFXQfX4Kk+\nURAHBHlwP5dv2LwxocaLAAAAHHF1ZXh0ZW5ATWFjQm9vay1Qcm8tMTYubG9jYWwB\n-----END OPENSSH PRIVATE KEY-----",
```

Comment:
I suggest changing this to B64 encoded PKCS8 DER which I believe matches passkeys.